1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,6 +17,15 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
+    let i = 0;
+
+    while (i < birthdays.length - 1) {
+        let birthdayCheck = birthdays[i];
+        if (birthdayCheck.startsWith("July")) {
```

Comment:
How about using 'includes' method?

Reply:
.starts with July! Always learning!  I have used the .includes method but it is nice to see other ways of doing it